nutrition_core.py

```
@@ -1,30 +1,35 @@
 Supported BMR formulas:
 - Mifflin-St Jeor (most accurate for general population)
 - Harris-Benedict (traditional formula)
 - Katch-McArdle (for athletes with known body fat percentage)
 
-Activity levels based on Physical Activity Level (PAL) factors.
+Note: Activity level calculations and higher-level convenience functions
+(calculate_all_bmr, calculate_all_tdee) are delegated to core.metabolism module
+to avoid circular imports and ensure consistency.
 """
 
 from typing import Dict, Literal, Union
```

Flake8 (7.3.0)

[error] 'typing.Union' imported but unused

(F401)